162 2118749 proxy disconnected (#391)
infrawatch/documentation: doc-Service-Telemetry-Framework/modules/con_support-for-stf.adoc

> Red Hat supports the core Operators and workloads, including {MessageBus}, AMQ Certificate Manager, Service Telemetry Operator, and Smart Gateway Operator. Red Hat does not support the community Operators or workload components, inclusive of ElasticSearch, Prometheus, Alertmanager, Grafana, and their Operators.

Reviewer: ```suggestion
Red Hat supports the core Operators and workloads, including {MessageBus}, AMQ Certificate Manager, Service Telemetry Operator, and Smart Gateway Operator. Red Hat does not support the community Operators or workload components, such as ElasticSearch, Prometheus, Alertmanager, Grafana, and their Operators.
```

> {ProjectShort} works only in a fully connected network environment. {ProjectShort} does not work in Red Hat OpenShift Container Platform-disconnected environments or network proxy environments.

Reviewer: ```suggestion
You can only deploy {ProjectShort} a fully connected network environment. You cannot deploy {ProjectShort} in Red Hat OpenShift Container Platform disconnected environments or network proxy environments.
```
